tools/system_libs.py

```diff
@@ -932,6 +932,9 @@ def get_cflags(self):
       cflags.append('-D_LIBCXXABI_NO_EXCEPTIONS')
     elif self.eh_mode == Exceptions.EMSCRIPTEN:
       cflags.append('-D__USING_EMSCRIPTEN_EXCEPTIONS__')
+      # The code used to interpret exceptions during terminate
+      # is not compatible with emscripten exceptions.
+      cflags.append('-DLIBCXXABI_SILENT_TERMINATE')
```

Comment:
some of these changes appear in another PR, which I guess should land first?

Reply:
I'm not sure what happened here.   This change was based on #14430 which is now landed.  Maybe i didn't set the correct upstream branch in the UI.